### PR TITLE
Fix square holistic corners

### PR DESCRIPTION
**Pull request by devpow112**
Fxies issue I noticed with the corner cells of holistic rubrics. Fix doesn't effect non-holistic rubrics.

| **Before** | **After** |
| ----------- | ----------|
| before | after |

### CODE REVIEW

File: src/components/editor/rubric-criterion-editor.js [hunk 1]
```diff
@@ -331,7 +331,7 @@ class RubricCriterionEditor extends composeMixins(
 	render() {
 		return html`
 			<div class="criterion-wrap">
-				<div class="criterion-fields${this.isLast && this.context?.isPartialEditorEnabled ? ' -last' : ''}">
```
Comment (devpow112) [inline src/components/editor/rubric-criterion-editor.js:334]: The `this.isLast` variable doesn't exist anywhere in rubrics.

**Review by MitchelPaulin — APPROVED** [review]

File: src/components/editor/rubric-overall-levels-editor.js [hunk 2]
```diff
@@ -381,7 +381,7 @@ export class RubricOverallLevelsEditor extends composeMixins(
 					.token="${this.token}"
 					aria-label-langterm="rubricEditorAccessibility:overallDescriptionAriaLabel"
 					.context="${this.context}"
-					?first-and-corner="${this._isFirstAndCorner(i, this._overallLevels.length)}"
+					?first-and-corner="${this._isFirstAndCorner(i)}"
```
Comment (devpow112) [inline src/components/editor/rubric-overall-levels-editor.js:384]: The `this._overallLevels.length` isn't used by `this._isFirstAndCorner`.

**Review by mpharoah-d2l — DISMISSED** [review]

**Comment by devpow112** [comment]
Hmmmm, apparently my `this.isLast` to `this.LastRow` broke the partial editor view. Will try to figure out.

File: src/components/editor/rubric-criterion-editor.js [hunk 3]
```diff
@@ -331,7 +317,7 @@ class RubricCriterionEditor extends composeMixins(
 	render() {
 		return html`
 			<div class="criterion-wrap">
-				<div class="criterion-fields${this.isLastRow && this.context?.isPartialEditorEnabled ? ' -last' : ''}">
```
Comment (devpow112) [inline src/components/editor/rubric-criterion-editor.js]: Before when this was `this.isLast` it never added the `-last` class. Adding it breaks the way the rubric looks. Removing class selection entirely.

**Comment by devpow112** [comment]
VDiff changes this time were expected. They fixed the corner being obscured on holistic rubrics.

**Review by MitchelPaulin — APPROVED** [review]

**Pull request merged**